plugins/formal-verification/src/org/jetbrains/kotlin/formver/conversion/FreshNames.kt

```diff
@@ -26,3 +26,12 @@ data class SpecialFieldName(val name: String) : MangledName {
         get() = "special\$$name"
 }
 
+data object FunctionObjectName : MangledName {
```

Comment:
I think this name doesn't quite represent what this stands for anymore, since this is the method for calling a function object, not the name of a function object, right?